addons/modules/XEH_preInit.sqf

```diff
@@ -6,4 +6,15 @@ PREP_RECOMPILE_START;
 #include "XEH_PREP.hpp"
 PREP_RECOMPILE_END;
 
+["ModuleCurator_F", "Init", {
+    params ["_logic"];
+
+    _logic addEventHandler ["CuratorObjectPlaced",
+    {
```

Comment:
Same line.

Reply:
Damn, still need to get used to K&R style.